docs/modules/ROOT/pages/explanations/odoo-category.adoc

```diff
@@ -0,0 +1,45 @@
+= Effects Of Invoice Category Parameters
+
+Odoo lets users customize the PDF render output using categories ("sections") in invoices.
+Each category has the following additional properties:
+
+sequence::
+controls how line items are grouped together and how each category is ordered within an invoice.
+
+pagebreak::
+causes the next line item to be on the next page in a PDF (after the last line item).
+
+separator::
+causes a line printed with `\***` after the last line item within the same category.
+
+subtotal::
+causes an additional line printed with an accumulated subtotal of all line items within the same category.
+
+== Sequence Number
+
+If there are line items with sections defined, the rendering of an invoice is influenced by their section configuration.
+
+=== Unique Sequence Numbers
+
+If no two categories in an invoice share the same sequence number, then each line item gets grouped together when rendered in a PDF regardless of their position in the invoice data record. However, the ordering of the line items _within_ the same category is still determined by the ordering in the invoice record.
```

Comment:
```suggestion
If no two categories in an invoice share the same sequence number, then each line item gets grouped together regardless of their position in the invoice data record. However, the ordering of the line items _within_ the same category is still determined by the ordering in the invoice record.
```

For me the `when rendered in a PDF` makes the sentence harder to read and is not necessary from the context